windows/src/developer/kmndebug/version.rc

```diff
@@ -0,0 +1,30 @@
+1 VERSIONINFO
+  FILEVERSION		15,0,78,0
+  PRODUCTVERSION	15,0,78,0
+  FILEFLAGSMASK		0x3fL
+  FILEFLAGS		0x0L
+  FILEOS		0x4L
+  FILETYPE		0x2L
+  FILESUBTYPE		0x0L
+  BEGIN
+    BLOCK "StringFileInfo"
+    BEGIN
+      BLOCK "0C0904E4"
+      BEGIN
+        VALUE "CompanyName", "SIL International\0"
+        VALUE "FileDescription", "Keyman KMN Debug Library\0"
+        VALUE "FileVersion", "15.0.78.0\0"
+        VALUE "InternalName", "KMNDEBUG\0"
+        VALUE "LegalCopyright", "� SIL International\0"
```

Comment:
```suggestion
        VALUE "LegalCopyright", "\xA9 SIL International\0"
```

Reply:
Good catch. I will fix the version.rc issues in #6022 given there are other updates to version.rc there anyway!